Extensions/3D/Cube3DRuntimeObject.ts

```diff
@@ -396,6 +408,28 @@ namespace gdjs {
     shouldUseTransparentTexture(): boolean {
       return this._shouldUseTransparentTexture;
     }
+
+    _convertMaterialType(
+      materialTypeString: string
+    ): gdjs.Cube3DRuntimeObject.MaterialType {
+      if (materialTypeString === 'StandardWithoutMetalness') {
+        return gdjs.Cube3DRuntimeObject.MaterialType.StandardWithoutMetalness;
+      } else {
+        return gdjs.Cube3DRuntimeObject.MaterialType.Basic;
+      }
+    }
+
+    setMaterialType(materialTypeString: string) {
+      this._materialType = this._convertMaterialType(materialTypeString);
```

Comment:
nit: maybe if the new materialType is the same, don't do the update? This could save performance in some games by avoiding to update the face UV/materials 60 times a second.

Reply:
ok, even though it's not exposed with an action.